templates/base.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE "html">
```

Comment:
The DOCTYPE declaration should be `<!DOCTYPE html>` instead of `<!DOCTYPE "html">`. The quotes are unnecessary and incorrect for the DOCTYPE declaration.